### DISCUSSION TITLE
Download Attachment Image

### DISCUSSION THREAD

**Discussion by Athik321** [Budibase/budibase#14181] [Help] [answered]
The below  image shows my attachment list, where I have added a Download button. When I click on the Download button, I want to download the corresponding image. What should I do to achieve this?

![image](https://github.com/user-attachments/assets/79df3a67-0298-4b01-9ae3-71f0f9ba79d1)

Can we display the image in a table format with the image in one column and the name in the other column? The below image is my table where I store the image.

![image](https://github.com/user-attachments/assets/ca030d26-5eeb-4cdb-865f-94115060e197)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Athik321,

There is a download file action. You can read up on this on our documentation [here](https://docs.budibase.com/docs/data-actions#download-file). You can either supply the URL of the file. Or you can select the column and supply the row ID and it will download the file that way.

I hope this helps.
